README.md

```diff
@@ -33,3 +33,11 @@ import TagInput from 'react-native-tag-input';
 | maxHeight | Max height of the tag input on screen (will scroll if max height reached) |
 | onHeightChange | Callback that gets passed the new component height when it changes |
 | parseOnBlur | Whether to treat a blur event as a separator entry (iOS-only) |
+| parseOnSubmit | Whether to treat submit button press as a separator entry |
+| onChangeText | A handler to be called when input text has changed |
+| scrollHorizontal | Whether the wrapper scrollView should scroll horizontally |
+| scrollViewProps | Any misc. ScrollView props (showsHorizontalScrollIndicator, etc.) |
+
+| Public Useful Method | Description |
+-----------------------|-----------------
+| addCustomTag | Replace pending input with suggested custom tag |
```

Comment:
You forgot to document "parseTags", which you are using from the example. That said, I don't think we should have any "public useful methods", and should just raise the `text` state.

Reply:
👍